app/celer-g4/RootIO.cc

```diff
@@ -54,6 +54,11 @@ RootIO::RootIO()
         file_name_ = "celer-g4.root";
     }
 
+    if (file_name_ == "-")
+    {
+        file_name_ = "stdout-" + std::to_string(::getpid()) + ".root";
```

Comment:
I'm sure ROOT leaks the include, but this requires `#include <unistd.h>` on POSIX machines... does it require anything on windows?

Reply:
It is the (misnamed) `_WIN32`.